src/MirOS.xml

```diff
@@ -0,0 +1,92 @@
+<SPDX name="MirOS Licence" identifier="MirOS" osi-approved="true">
```

Comment:
agreed - that is how it appears on OSI site too, not sure how that 'c' got in there.